packages/webapp/pages/[userId]/index.tsx

```diff
@@ -252,6 +251,22 @@ const ProfilePage = ({ profile }: ProfileLayoutProps): ReactElement => {
               />
             )}
           </ActivityContainer>
+          <ActivityContainer>
+            <ActivitySectionTitle>
+              Top tags by reading days
+            </ActivitySectionTitle>
+            <div className="grid grid-cols-1 tablet:grid-cols-2 gap-3 tablet:gap-10">
+              {readingHistory.userReadingTopTags?.map((topTag) => (
+                <ReadingTagProgress
+                  key={topTag.tag}
+                  topTag={topTag}
+                  isFilterSameYear={
+                    dropdownOptions[selectedHistoryYear] === CURRENT_YEAR
+                  }
```

Comment:
Why does this matter?

Reply:
For example, you selected 2022 and we only have 32 days yet - the label would be "7/32 reading days"

Reply:
Cant we just always get the max days for the selected year then?

Reply:
Totally cool. Please have some rest! You've been grinding the fest! 💪